### PR TITLE
Exclude more flaky tests from Mac workflows

### PR DESCRIPTION
**Pull request by bduffany**
---

**Version bump**: Patch

### CODE REVIEW

File: buildbuddy.yaml [hunk 1]
```diff
@@ -25,6 +25,8 @@ actions:
         test --config=mac-workflows
         --
         //...
+        -//enterprise/server/raft/store:all
```
Comment (maggie-lou) [inline buildbuddy.yaml:28]: Why are we just excluding the flaky tests for mac?

Comment (bduffany) [reply buildbuddy.yaml:28]: These tests were previously excluded and recently re-enabled; I am just re-disabling them. We can probably increase the timeouts or re-enable them at some point, but I figured it's not worth thinking about right now.

Comment (bduffany) [reply buildbuddy.yaml:28]: Actually, I didn't realize I hadn't submitted https://github.com/buildbuddy-io/buildbuddy/pull/3632 . Let's merge that one first and see if it helps

Comment (bduffany) [reply buildbuddy.yaml:28]: It didn't help - reopening.

Comment (maggie-lou) [reply buildbuddy.yaml:28]: It still says it's closed for me so I can't approve it

**Pull request closed without merging**